__tests__/pet.test.js

```diff
@@ -30,7 +31,7 @@ describe('growUp', () => {
     });
 });
 
-
+// you don't need a describe for each method - the two `it` calls below can be in one describe
```

Comment:
You have a lot of duplicate describe blocks in your test file. You can nest multiple tests inside a describe callback instead of this duplication:

```js
describe('my method', () => {
  it('does something', () => {});

  it('does something else', () => {});
});
```